src/actions.ts

```diff
@@ -654,6 +654,22 @@ export const actions: {[k: string]: QueryHandler} = {
 		await tables.oauthTokens.deleteAll()`WHERE client = ${client.id} and owner = ${this.user.id}`;
 		return {success: true};
 	},
+
+	async getteams(params) {
+		if (!this.user.loggedIn || this.user.id === 'guest') {
+			return {teams: []}; // don't wanna nag people with popups if they aren't logged in
+		}
+		let teams = [];
+		try {
+			teams = await tables.teams.query(
+				'SELECT * FROM teamid, team, format, title as name WHERE ownerid = $1', [this.user.id]
+			) || [];
```

Comment:
Perhaps `??` instead. Also I don't actually know the precedence of `await` vs `??`, is this doing what you want it to do?

Reply:
Tested, looks like await comes first. Updated.